code/modules/SCP/SCPs/SCP-2427-3.dm

```diff
@@ -10,10 +10,10 @@ GLOBAL_LIST_EMPTY(scp2427_3s)
 	attack_verb = list("stabbed")
 	hitsound = 'sound/scp/2427/stab.ogg'
 	damtype = BRUTE
-	melee_accuracy_bonus = 200
+	melee_accuracy_bonus = 60 //skull emoji
```

Comment:
Please get rid of unnecessary comments, and remember that comments you make will be viewed without the context of what the pr changed, so anything relating to previous values should also not be included.